Dockerfile

```diff
@@ -1,20 +1,60 @@
-FROM openjdk:11
+ARG OVERWRITE_VERSION
+
+### Stage 1: build UI
+FROM node:12 AS BUILD_UI_STAGE
+
+WORKDIR /agr_curation
+COPY src/main/cliapp ./cliapp
+COPY src/main/resources ./resources
+
+WORKDIR /agr_curation/cliapp
+RUN make all build
+
+
+
+### Stage 2: build API (and include UI components)
+FROM maven:3.8-openjdk-11 as BUILD_API_STAGE
+ARG OVERWRITE_VERSION
+
+# copy the pom and src code to the container
+COPY ./ ./
+COPY --from=BUILD_UI_STAGE /agr_curation/resources ./src/main/resources
+
+# Install make
+RUN apt-get update -qq && \
```

Comment:
For instance this happened to use with the Apollo build a couple of releases ago. Couldn't release production until `apt-get update` finally got fixed.

Reply:
@oblodgett I can't install `make` without running `apt-get update` first, otherwise the install fails. That being said, as the `maven:3.8-openjdk-11` image runs Debian 11 rather than Ubuntu 18.04 (which is used for the Apollo image), I assume `apt-get update` would be much more reliable than in Ubuntu, so I don't believe this would be much of a problem.

Reply:
Can we just run the commands directly in the dockerfile, this eliminates the external dependency which at some point will cause build failures.